python/private/whl_target_platforms.bzl

```diff
@@ -118,68 +103,129 @@ def _whl_priority(value):
     # Windows does not have multiple wheels for the same target platform
     return (False, False, 0, 0)
 
-def select_whl(*, whls, want_abis, want_platform):
-    """Select a suitable wheel from a list.
+def select_whls(*, whls, want_version = "3.0", want_abis = [], want_platforms = []):
+    """Select a subset of wheels suitable for target platforms from a list.
 
     Args:
         whls(list[struct]): A list of candidates.
+        want_version(str): An optional parameter to filter whls by version. Defaults to '3.0'.
```

Comment:
Is this version referring to python version? If so, please clarify in the doc or arg name.